build.gradle

```diff
@@ -234,7 +234,7 @@ android {
     }
 
     // See: https://developer.android.com/studio/build/build-variants#flavor-dimensions
-    flavorDimensions += ["app", "recents", "channel"]
+    flavorDimensions += ["app", "recents", "channel", "accessibility"]
```

Comment:
You don't have to add a new flavor, just place accessibility related code under `github` falvor.

Reply:
agree since the market is for playstore all along

Reply:
Good point- I forgot that we do not release on any other place other than these two...

Reply:
Should be OK now